src/modules/DialogOverlay.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <Dialog
+    v-if="isDialogOpen"
+    id="widget-dialog"
+    :button-close-label="buttonCloseLabel"
+    @toggle-dialog="toggleDialog"
+  >
+    <template #header> <slot name="heading"></slot> </template>
+    <template #body>
+      <slot name="tabs"></slot>
+      <slot name="terms"></slot>
```

Comment:
import tabs and accordian components in.
Pass data as props into tabs and accordian